.circleci/config.yml

```diff
@@ -244,14 +250,22 @@ workflows:
       - test:
           requires:
             - build
+      - dps/helm_lint:
+          name: helm_lint_dev
+          env: development
+          chart_name: sentence-planning
```

Comment:
You can take this param out if you like, as the orb defaults this value using `'${CIRCLE_PROJECT_REPONAME}'`

Reply:
i put this in on purpose because the circleci local CLI doesn't populate this var for local testing. figured it wouldn't do any harm to leave it there :)